sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -19,6 +19,18 @@
 
 #include <_az_cfg_prefix.h>
 
+
+/**
+ * @brief Azure IoT service MQTT bit field properties for telemetry publish messages.
+ *
+ */
+enum
+{
+  AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_QOS = 0,
+  AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_DUPLICATE = 0,

```

Comment:
I'm not sure about documenting the Duplicate =  0 given that is an MQTT implementation detail: first message must have DUP set to 0. A retry should change that to 1.